home.html

```diff
@@ -66,29 +75,29 @@ <h2>2023.02.01(WED) ~ 02(THU)</h2>
             <h3>Main Program</h3>
             <hr class="orangehr">
             <ul class="programcards">
-                <li class="minicard"><img src="./imgs/powerpoint-128.png"><span class="redfont">Presentation</span>
+                <li class="minicard"><img alt="powerpoint" src="./imgs/powerpoint-128.png"><span class="redfont">Presentation</span>
                     <p>
                         Listen to our professionals explanation on what we plan for now on.
                     </p>
                 </li>
-                <li class="minicard"><img src="./imgs/article-marketing-128.png"><span class="redfont">Exhibition</span>
+                <li class="minicard"><img alt="marketing" src="./imgs/article-marketing-128.png"><span class="redfont">Exhibition</span>
                     <p>
                         Observe and interact with our successful subjects for a closer view of what we crave.
                     </p>
                 </li>
-                <li class="minicard"><img src="./imgs/speech-bubble-5-128.png"><span class="redfont">Forum
+                <li class="minicard"><img alt="speech" src="./imgs/speech-bubble-5-128.png"><span class="redfont">Forum
       </span>
                     <p>
                         Have the time to share your thoughts and opinions with experts on the topic.
                     </p>
                 </li>
-                <li class="minicard"><img src="./imgs/analytics-128.png"><span class="redfont">Workshop
+                <li class="minicard"><img alt="analytics" src="./imgs/analytics-128.png"><span class="redfont">Workshop
              </span>
                     <p>
                         An oportunity to see with your own eyes the work of our professionals.
                     </p>
                 </li>
-                <li class="minicard"><img src="./imgs/dollar-2-128.png"><span class="redfont">CGI Start
+                <li class="minicard"><img alt="dollar" src="./imgs/dollar-2-128.png"><span class="redfont">CGI Start
   </span>
                     <p>
```

Comment:
## [OPTIONAL]

- [ ] If we just check the original design we will find out that there is a hover effect on each program card. I think kindly we can improve this design by adding this effect.
![image](https://user-images.githubusercontent.com/65068771/148632527-ac785523-d80e-4048-929e-82d6db218856.png)